src/Schema/Events/Click.ts

```diff
@@ -418,3 +418,29 @@ export interface ClickedAppDownload {
   destination_path: string
   subject: string
 }
+
+/**
+ * A user clicks the 'Add to Calendar' button on a timed-event page (auction, fair, etc.).
+ *
+ * This schema describes events sent to Segment from [[ClickedAddToCalendar]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "clickedAddToCalendar",
+ *    context_module: "auctionHome",
+ *    context_page_owner_type: "sale",
+ *    context_page_owner_id: "5f99e0ba4c24bc000d02b8d7",
+ *    context_page_owner_slug: "artsy-x-seoul-auction-contemporary-icons",
+ *    subject: "google"
+ *  }
+ * ```
+ */
+export interface ClickedAddToCalendar {
```

Comment:
Considering that we also have this feature in iOs, this could be a good candidate to make agnostic to `click` or `tap` actions and go with a more generic `AddToCalendar`, similar to how the `Follow` event is setup: https://cohesion.artsy.net/interfaces/_schema_events_savesandfollows_.followedartist.html 

For events that can be shared between apps we are also using the generic `context_owner_type` rather than `context_screen`... or `context_page`... 

cc @abhitip

Reply:
I don't think that's a bad idea! The events will still end up in separate schemas/tables in Redshift `force_production` vs `eigen_production`. Making it one consistent schema will make it easier to analyze. What do you think @mikehrom?

Reply:
Agree 👍  will update